solidity/contracts/Staking.sol

```diff
@@ -400,4 +400,20 @@ contract Staking {
         return (interestData.globalTotalStaked, totalReward, estimatedReward, unlockedReward, accruedReward);
 
     }
+
+    /**
+    * @dev staken token getter.
```

Comment:
should be a sentence that describes the function,  like: "Returns stake token address"